website/content/docs/sync/gcpsm.mdx

```diff
@@ -160,8 +155,9 @@ to provision a GCP Service Account with the appropriate policies.
       // GOOGLE_REGION (optional)
       // GOOGLE_PROJECT
       // GOOGLE_CREDENTIALS	(The path to a service account key file with the
-      //				        "Service Account Admin", "Service Account Key
-      //					    Admin", and "Security Admin" roles attached)
+      //				        "Service Account Admin", "Service Account Key Admin",
+      //                "Secret Manager Admin", and "Project IAM Admin" roles
+      //                attached)
```

Comment:
Is it missing 1 tab on the last 2 lines for the comment to be aligned as a block? Was it the purpose?